engine/apps/alerts/tasks/send_update_log_report_signal.py

```diff
@@ -7,7 +7,7 @@
 
 
 @shared_dedicated_queue_retry_task(
-    autoretry_for=(Exception,), retry_backoff=True, max_retries=1 if settings.DEBUG else None
+    autoretry_for=(Exception,), retry_backoff=True, max_retries=1 if settings.DEBUG else 10
```

Comment:
it is safe to set maximum retries there since the only place where the task can get exception is getting alert group.